src/ansys/fluent/core/filereader/casereader.py

```diff
@@ -78,6 +81,8 @@ class CaseReader:
 
     def __init__(self, case_filepath: str):
 
+        if Path(case_filepath).suffix in [".flprj", ".flprz"]:
```

Comment:
@prmukherj I would prefer it if a project file path were indicated by a separate argument rather than case_filepath having multiple meanings.

Reply:
Actually, the approach being followed here is that if the provided path is project file path then it'll extract the case file path and carry on or else the provided path might be a case file path.
So, shall we ask the user to input either a case file-path or project-file path?

Reply:
@prmukherj I had followed the approach; I just though it preferable not to bend the meanings of arguments. If reusing the same general argument, then I would prefer to change it's name to something generic (just filepath), but I prefer it if we are explicit about the meanings of things (i.e. have two distinct arguments).